grails-app/controllers/au/org/ala/merit/ManagementUnitController.groovy

```diff
@@ -262,23 +262,15 @@ class ManagementUnitController {
         render model:cmd.model, view:'/activity/activityReportView'
     }
 
-    @PreAuthorise(accessLevel = 'readOnly', redirectController = 'managementUnit')
-    def reportPDF(String id, String reportId) {
-
```

Comment:
You've essentially copy/pasted a whole bunch of code here - the method should look more like viewOrganisationReport with a new PrintOrganisationReportCommand modelled off how the ViewOrganisationReportCommand looks (using the PRINT mode).

The idea of moving to the command design is to allow the common code for viewing reports to be reused across MU/Program/Org/Project rather than having essentially a cut and paste in each controller.